index.d.ts

```diff
@@ -0,0 +1,129 @@
+// Type definitions for node-vault 0.10
```

Comment:
There is no 0.10 version at the moment. Please remove version from comment.

Reply:
Resolved. @kr1sp1n Thanks for taking a look into this!